src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -230,12 +230,13 @@ public object ExplicitDefaultValue
         /// </remarks>
         internal abstract ConstantValue ExplicitDefaultConstantValue { get; }
 
-        internal override bool NullableOptOut
+        internal override bool NonNullTypes
         {
             get
             {
+                Debug.Assert(false); // PROTOTYPE(NullableReferenceTypes): Can we reach this?
```

Comment:
>// PROTOTYPE(NullableReferenceTypes): Can we reach this? [](start = 37, length = 56)

Perhaps: `F([NotNullTypes]List<object> arg = new List<object>())`

Reply:
I still didn't manage to reach this. I'll keep PROTOTYPE for a follow-up.

---
In reply to: [195588470](https://github.com/dotnet/roslyn/pull/27807#discussion_r195588470) [](ancestors = 195588470)